src/manifolds/Grassmann.jl

```diff
@@ -215,7 +215,7 @@ g_p(X,Y) = \operatorname{tr}(X^{\mathrm{H}}Y),
 
 where $\cdot^{\mathrm{H}}$ denotes the complex conjugate transposed or Hermitian.
 """
-inner(::Grassmann, p, X, Y) = dot(X, Y)
+inner(::Grassmann, p, X, Y) = dot(abs.(X), abs.(Y))
```

Comment:
I thought we inherited the metric from the embedding, and this change differs from manopt's implementation. Can you explain this?

Reply:
Oh, sorry the abs - es are from some checks, they were left there by error.